src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -34,7 +39,8 @@ public class SwerveSubsystem implements Subsystem {
   public SwerveModule[] swerveModules = new SwerveModule[4];
   public final Vector[] vectorKinematics = new Vector[4];
   private final SwerveKinematics swerveKinematics;
```

Comment:
Can we delete "our versions" - the SwerveKinematics and ChassisVelocity classes? They shouldn't be used anymore.